buildmachine/add-users-to-docker-group.ps1

```diff
@@ -0,0 +1,31 @@
+#Requires -RunAsAdministrator
+
+[CmdletBinding()]
+param (
+	[Parameter()][string]$UserName,
+	[Parameter()][int]$NumberOfUsers,
+	[Parameter()][int]$StartUserNumber = 1
+)
+
+$userNameMaxLength = 15
+if ([string]::IsNullOrWhiteSpace($UserName)) {
+	$UserName = Read-Host -Prompt 'User name'
+}
+if (-not ($UserName -match "^[a-z0-9_-]{1,$userNameMaxLength}$")) {
+	throw "Invalid user name. It must be a string containig only letters, numbers, underscore (_) and hyphen (-) and its maximum length can be $userNameMaxLength characters."
+}
+if ($NumberOfUsers -lt 1) {
+	$NumberOfUsers = [int](Read-Host -Prompt 'Number of users')
+	if ($NumberOfUsers -lt 1) {
```

Comment:
Viem že je to asi skopírované, ale možno takéto ručné zdávanie nepotrebujeme. Stačí spraviť parameter `NumberOfUsers` povinný a ak nie je zadaný, v interaktívnom režime sa Powershell sám naň spýta. Ak človek explicitne zadá 0, tak proste zadal nulu a nič sa nestane.